SEO/Controllers/VisitController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Web.Http;
+using SEO.Models;
+using DataBase.Models.Visitors;
+using DataBase.DAL;
+using SEO.BLL;
+using MongoDB.Driver;
+
+namespace SEO.Controllers
+{
+    public class VisitController : ApiController
+    {
+
+        private readonly DatabaseContext dataBase = new DatabaseContext();
+        ValidUser visit = new ValidUser();
+        // GET: api/ValidVisit
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET: api/ValidVisit/5
+        public string Get(int id)
+        {
+            return "value";
+        }
+
+        // POST: api/ValidVisit
+        public async Task Post([FromBody]VisitInfo visitInfo)
+        {
+            var builderUser = Builders<Visitor>.Filter;
```

Comment:
Controller should not know anything about builders and other MongoDB stuff. It should only operate with VisitorManager